hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -225,3 +282,85 @@ rpi-eeprom-config --out pieeprom-netboot.bin --config bootconf.txt pieeprom.bin
 sudo rpi-eeprom-update -d -f ./pieeprom-netboot.bin
 sudo reboot
 ```
+
+## USB mass storage boot
+This is only available in the BETA release and requires updated (not yet released) firmware via rpi-update. If you aren't already familiar with how to use a USB drive for the root filesystem then you probably want to wait until this feature is in the default Raspbian image.
+
+There's no support for migrating a SD card image to a USB drive. It's possible but the process can potentially be quite involved and varies according to your original setup. Please see [STICKY: HOWTO: Move the filesystem to a USB stick/Drive](https://www.raspberrypi.org/forums/viewtopic.php?f=29&t=44177&start=350)
+
+## BETA setup instructions
+These instructions assume that you are familiar with manual firmware and bootloader updates and understand how to revert to a previous version if you want to revert the changes. If not, please wait until the features are available in a full Raspbian release image.
+
+There will be a post on the Raspberry Pi [General Discussion forum](https://www.raspberrypi.org/forums/viewforum.php?f=63) when the binaries are available.
```

Comment:
MIght be worth linking directly to the forum post, once this is all live obviously.

Reply:
When this is live which is after this and the packages are released etc

Reply:
Sorry, yes that's what I was meaning by "this all" 😋